test/Microsoft.AspNet.Mvc.Core.Test/Builder/MvcApplicationBuilderExtensionsTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Moq;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Core.Builder
+{
+    public class MvcApplicationBuilderExtensionsTest
+    {
+        [Fact]
+        public void UseMvc_ThrowsInvalidOperationException_IfRoutingMarkerServiceIsNotRegistered()
+        {
+            // Arrange
+            var applicationBuilderMock = new Mock<IApplicationBuilder>();
+            applicationBuilderMock
+                .Setup(s => s.ApplicationServices)
+                .Returns(Mock.Of<IServiceProvider>());
+
+            // Act
+            var error = Assert.Throws<InvalidOperationException>(
```

Comment:
May be worthwhile being consistent with most similar tests: Use `exception` or `ex`. `error` doesn't mean anything here.